examples/graphql_ws_subscription_example.py

```
@@ -0,0 +1,70 @@
+async def ticker_source(*_, **kwargs: dict[str, Any]):
+    """
+    Async generator emitting a counter number every second until limit is reached.
+    """
+    limit = kwargs.get("limit")
+    for _counter in range(limit):
+        yield _counter
+        await asyncio.sleep(1)
+
+
+@subscription.field("ticker")
+def resolve_ticker(value, *args, **kwargs):
```

Ruff (0.15.2)

[warning] Unused function argument: `args`

(ARG001)